### PR TITLE
Update data test ids for 3.1

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I updated data test ids for 3.1

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5141 Update test attributes names](https://app.clickup.com/t/2549495/SALEOR-5141)

**Review by orzechdev — APPROVED** [review]

**Review by eCzerniak — CHANGES_REQUESTED** [review]
Please check your PR because some tests are failing and this is connected with some fronted problems. Tests look ok.

File: cypress/integration/products/productsList/pagination.js [hunk 1]
```diff
@@ -16,10 +16,11 @@ filterTests({ definedTags: ["all"] }, () => {
       cy.clearSessionData().loginUserViaRequest();
       cy.visit(urlList.products);
     });
+
     it("Should go to the next page", () => {
-      cy.softExpectSkeletonIsVisible();
-      cy.get(PRODUCTS_LIST.productsList)
-        .should("be.visible")
+      cy.softExpectSkeletonIsVisible()
+        // cy.get(PRODUCTS_LIST.productsList)
```
[the remark this answers is not in the document]
Comment (karola312) [reply cypress/integration/products/productsList/pagination.js]: I removed comment

**Review by eCzerniak — APPROVED** [review]

**Comment by eCzerniak** [comment]
Could you please change API_URI to master staging?

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**